pxf/build.gradle

```diff
@@ -266,10 +273,7 @@ project('pxf-service') {
             into '/opt/pivotal/pxf/tomcat-templates'
```

Comment:
line 259 -- dropping pxf-service into /etc/init.d would that conflict with side-by-side install ? should the script inside /etc/init.d be also a symlink ?